.runpod/README.md

```diff
@@ -0,0 +1,335 @@
+<h1>LLM Training- Full finetune, LoRA, QLoRa etc. Llama/Mistral/Gemma</h1>
+
+# Configuration Options
+
+This document outlines all available configuration options for training models. The configuration can be provided as a JSON request.
+
+## Usage
+
+You can use these configuration Options:
+
+1. As a JSON request body:
```

Comment:
I noticed this is the only item in a numbered list. Do you have other usage options or can we remove the numbering?

Also, is there an easy way (e.g., a script) to create this request body from a config YAML file?